### PR TITLE
Add dctmd tools, part 2

### PR DESCRIPTION
**Pull request by simonbray**

### CODE REVIEW

File: tools/biomoldyn/NEQGamma.xml [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+            -av '$av'
+            -sigma '$sigma'
+            #if $json:
+                -json '$json'
+            #end if
+
+    ]]></command>
+    <inputs>
+        <param type="data_collection" name="xvgs" label="XVG constraint force files" format="xvg" help="XVG files generated by GROMACS"/>
+        <param type="data" name="json" optional="true" label="Ensemble subgroups" format="json" help="Optional: list of sub-ensembles in JSON format (e.g. produced by the 'Extract clusters' tool). If not specified, results will be calculated for the entire ensemble."/>
```
Comment (bgruening) [inline tools/biomoldyn/NEQGamma.xml:32]: @OlegZharkov more json files. A nice default standard view would be really cool I guess.

File: tools/biomoldyn/NEQGamma.xml [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+        <param label="Pull velocity in nm/ps of the TMD simulations" value="0.001" type="float" name="vel" argument="-vel"/>
+        <param label="Temperature in K" value="300" type="float" name="T" argument="-T"/>
+        <param label="Size of averaging window for displaying Gamma(x)" value="0" type="integer" name="av" argument="-av" help="Recommended: 4 to 20 per 100 data points."/>
```
Comment (bgruening) [inline tools/biomoldyn/NEQGamma.xml:35]: min=0?

Comment (simonbray) [reply tools/biomoldyn/NEQGamma.xml]: Yep, thanks, makes sense for all these values.

**Review by bgruening — APPROVED** [review]

**Comment by simonbray** [comment]
Sorry @bgruening, I cannot merge, would you mind?

**Comment by simonbray** [comment]
Thanks :+1:

**Pull request merged**